packages/all/package.json

```diff
@@ -30,30 +30,28 @@
     "build": "npx tsc -p ./config",
     "postbuild": "npx rollup -c ./config/rollup.config.js",
     "predeploy": "npm run build",
-    "deploy": "npm publish"
+    "deploy": "npm publish --tag beta"
   },
   "peerDependencies": {
-    "@types/react": "^16.8.0",
-    "apollo-client": "^2.6.4",
-    "graphql": "^14.3.1",
-    "react": "^16.8.0",
-    "react-dom": "^16.8.0"
+    "@types/react": "^16.9.17",
+    "graphql": "^14.5.8",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0"
+  },
+  "devDependencies": {
+    "rollup": "1.29.0",
+    "typescript": "3.7.4"
   },
   "dependencies": {
     "@apollo/react-common": "file:../common",
     "@apollo/react-components": "file:../components",
     "@apollo/react-hoc": "file:../hoc",
-    "@apollo/react-hooks": "file:../hooks",
```

Comment:
Why remove `@apollo/react-hooks` here? Don't we want to export hooks-related stuff from `react-apollo`?

Ah, I think I see now: since the exports of `@apollo/react-hooks` are a subset of the exports of `@apollo/react-common`, we don't need to import `@apollo/react-hooks` for the combined `react-apollo` package?

If that's accurate, I think I'd like to keep depending on `@apollo/react-hooks` here, just because the relationship between it and `@apollo/react-common` could change.

Reply:
Exactly, and no problem on adding the dependency back.